src/Sarif.Driver/Sdk/MultithreadedAnalyzeCommandBase.cs

```diff
@@ -236,24 +236,27 @@
             context.OutputFilePath = options.OutputFilePath;
             context.AutomationGuid = options.AutomationGuid;
             context.BaselineFilePath = options.BaselineFilePath;
-            context.Traces = InitializeStringSet(options.Trace);
+            context.Traces = options.Trace != null ? InitializeStringSet(options.Trace) : context.Traces;
```

Comment:
>Trace [](http://example.com/codeflow?start=37&length=5)

This pattern needs to be pushed through-out this code. 

If options return null consistently, then we can override an existing config object with updated values but ONLY if the user has expressed an override on the command-line.